Doc/library/stdtypes.rst

```diff
@@ -441,6 +440,11 @@ Notes:
    A right shift by *n* bits is equivalent to division by ``pow(2, n)`` without
    overflow check.
 
+(4)
```

Comment:
Forgot to add references to the new footnote?

Reply:
@vadmium How should I reference it? Is this what is required?

```
index 5fa3ca10fd..40129f0037 100644
--- a/Doc/library/stdtypes.rst
+++ b/Doc/library/stdtypes.rst
@@ -400,7 +400,7 @@ Bitwise Operations on Integer Types
    operator: >>

 Bitwise operations only make sense for integers.  Negative numbers are treated
-as their 2's complement value.
+as their 2's complement value (4).
```

Reply:
If you are trying to follow Nick’s suggestion (move the note about negative numbers to a numbered footnote in the table), I think he was suggesting adding (4) to the table that the other three footnotes relate to, specifically to each of the first three rows for x | y, x ^ y, and x & y. These are bitwise binary operations and they define _x_ and _y_.

However I am not enthusiastic about the new text; see my comments in the bug thread.

Reply:
So, what do you think about Nick's suggestion, where he included Mark's suggestion as well: 

``` Each bitwise operation has the same result as though carried out in two's complement using a bit-width that's large enough to represent the inputs. ("Large enough" for this purpose is ``1 + max(x.bit_length(), y
.bit_length()``, with the extra bit being needed to handle sign extension appropriately)`